docs/development/setup/wsl-rebuild.md

```diff
@@ -0,0 +1,33 @@
+If you ever want to recreate your development environment again from
+scratch (e.g., to test a change you've made to the provisioning
+process, or because you think something is broken), you can do so using the following steps:
+
+Deactivate the python virtual environment:
+
+```console
+$ deactivate
+```
```

Comment:
Is this step necessary? I'm not sure it has any effect -- just changes some shell variables, and the next step is not inside the container anyway.